source/extensions/dynamic_modules/abi.h

```diff
@@ -1234,6 +1311,26 @@ envoy_dynamic_module_callback_http_filter_http_callout(
     envoy_dynamic_module_type_buffer_module_ptr body, size_t body_size,
     uint64_t timeout_milliseconds);
 
+/**
+ * envoy_dynamic_module_callback_http_filter_continue_decoding is called by the module to continue
+ * decoding the HTTP request.
+ *
+ * @param filter_envoy_ptr is the pointer to the DynamicModuleHttpFilter object of the
+ * corresponding HTTP filter.
+ */
+void envoy_dynamic_module_callback_http_filter_continue_decoding(
+    envoy_dynamic_module_type_http_filter_envoy_ptr filter_envoy_ptr);
+
+/**
+ * envoy_dynamic_module_callback_http_filter_continue_encoding is called by the module to continue
+ * encoding the HTTP response.
+ *
+ * @param filter_envoy_ptr is the pointer to the DynamicModuleHttpFilter object of the
+ * corresponding HTTP filter.
+ */
+void envoy_dynamic_module_callback_http_filter_continue_encoding(
+    envoy_dynamic_module_type_http_filter_envoy_ptr filter_envoy_ptr);
```

Comment:
I think this two method should be scheduled to correct worker thread to run, right? If so, please add an obvious note or attention on the comment.

Reply:
yeah all functions accepting `envoy_dynamic_module_type_http_filter_envoy_ptr filter_envoy_ptr` is basically the same so i will end up adding that comment to all methods ... haha. But you comment is actually valid though, I think i add that comment to all functions in a follow PR.